lib/components/player.dart

```diff
@@ -1,4 +1,5 @@
 import 'dart:async';
+//import 'dart:html'; // TODO move to web only.
```

Comment:
web stuff should not be commented out

Reply:
@abegehr this is not used. what exactly should be done here? delete the line completely?

Reply:
this is used on web, but causes crashes on native. See #67